app/views/shared/_index_vendors.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% vendors.each do |vendor| %>
+    <% unless vendor.archived %>
```

Comment:
Like before, this is something you could handle at the model level with something like:
```ruby
def self.active_vendors
  where(archived: false)
end
```
And then just set vendors to that in your controller.